### ISSUE TITLE
Cannot select a time point with TimePickerDialog when selectableTimes has duplicate time points

### ISSUE THREAD

**Issue by kyukyukyu** [wdullaer/MaterialDateTimePicker#361]
I tried to set every 90 minutes from 12 AM as valid selections of my `TimePickerDialog`, so I wrote a code like this: (in Kotlin)

```kotlin
val selectableTimes = (0...(24 * 60) step 90)
            .map { Timepoint(it / 60, it % 60) }
            .toTypedArray()

// ...

val tpd = TimePickerDialog.newInstance(this, false)
tpd.setSelectableTimes(selectableTimes)
```

And yes, **I made a mistake to put 12 AM twice.** By the way, when I ran this code, my TimePickerDialog didn't work correctly. I couldn't pick any time point other than 12 AM. By replacing `0...(24 * 60) step 90` with `0 until (24 * 60) step 90`, I could make it work correctly so that I could pick every 90 minutes from 12 AM.

It seems TimePickerDialog cannot handle invalid `selectableTimes` at the moment. I think it would be great to get this problem fixed, or at least it should be documented that duplicate time points are not allowed.

**Comment by wdullaer** [comment]
I'll pick this up next time I revisit this code.

**Comment by wdullaer** [comment]
This should be fixed in v3.3.0
